SensorArray.cpp

```diff
@@ -0,0 +1,94 @@
+#include "SensorArray.h"
+#include "Sensor.h"
+
+
+using namespace std;
+
+namespace LFRobot
+{
+	SensorArray::SensorArray(int pins[])
+	{
+		Sensor* sensorArray[NUMBER_OF_SENSORS];
+
+		for (int i = 0; i < NUMBER_OF_SENSORS; i++)
+		{
+			sensorArray[i] = new Sensor(pins[i]);
+		}
+	}
+
+	SensorArray::~SensorArray()
+	{
+		for (int i = 0; i < NUMBER_OF_SENSORS; i++)
+		{
+			delete sensorArray[i];
+		}
+	}
+
+
+	void SensorArray::prepSensors()
+	{
+
+		//	Sensor sensorArray[8] = { Sensor(pins[0]), Sensor(pins[1]), Sens };
+
+
+		for (int i = 0; i < NUMBER_OF_SENSORS; i++)
+		{
+			sensorArray[i]->setMode(OUT);
+
+		}
+	}
+
+
+	void SensorArray::readSensorValues()
+	{
```

Comment:
If this ends up being too slow/inacurrate I thought of a really fast way using bit operations